docs/browser-engine/webview2.md

```diff
@@ -0,0 +1,72 @@
+---
+title: WebView2
+---
+
+# WebView2
+
+<!-- ✍ Please populate the content `/data/technologies/webview2.json`
+first. The text starting with `{{` will be interpolated in build time with
+the information in that file. -->
+
+WebView2 is one of the most popular technologies to build hybrid applications that leverage the best of both the native and web! With WebView2 you can embed html, css, and javascript to modernize your applications. This enables your app to have native app benefits like access to file manager and custom drm systems while getting the reach of the web. 
+
+WebView2 has recently GA'd but has been widely adopted by apps like Office, Outlook, Morgan Stanley, and many others. Watch how Morgan Stanley uses WebView2 [here](https://www.youtube.com/watch?v=8y3ZCzw3LtA).
+
+
+<!-- ✍ Please write about the technology, when it was created, what type of technology is it,
+some of the key characteristics for developers, etc. -->
+
+Althought the traditional IE WebBrowser control dates back to (2008 -- #JS can we find the actual date), WebView2 is a newer technology that has modernized the embeddable web browser by adopting the microsoft edge chromium platform. WebView2 has recently GA'd on Win32 C/C++, .NET, and expects to have UWP support soon. 
+
+WebView2 has both an evergreen and fixed distribution mode. With the default Evergreen mode, WebView2 will get the latest features, security, and bug-fixes automatically! For developers who want more control over their app updates we also offer the fixed version which allows you to ship webview2 with your app.
+
+WebView2 is backed by Microsoft, and continues to engage with the developer community and take feature requests and bug fixes.
+
+## Release cadence, version support and update model
+
+<!-- ✍ How often is there a new version? Is it predictable? Does it use an evergreen model (i.e.:
+it gets updated automatically) or do developers need to take care of that? Does it depend on other
+projects to ship? -->
+
+There are 2 parts to WebView2: the SDK, which comes in the form of a NuGet package, and the WebView2 Runtime, which take and optimize the edge browser binaries. The SDK and Runtime get updated every 4 weeks, in alignment with Edge. WebView2 offers a prerelease SDK with up and coming experimental features and a stable release SDK. 
+
+## Governance
+
+<!-- ✍ Please add an introductory paragraph about the governance model of the project: part of a
+foundation, company driven, etc. -->
+
+WebView2 is backed by Microsoft, and is built on top of the Edge browser platform. 
+
+## Summary
+
+| Characteristic |        |
+| -------------- | :----: |
+| Website | [{{technologies.webview2.url}}]({{technologies.webview2.url}}) |
+| Platforms | Desktop: Win32, WPF, WinForms |
```

Comment:
What do you think about skipping all the different type of Windows apps that can be done and just say the target is `Desktop`? I'd also add a note at the end saying that macOS is under development.

```suggestion
| Platforms | Desktop |
```